dex/testing/dcr/harness.sh

```diff
@@ -6,7 +6,7 @@ RPC_USER="user"
 RPC_PASS="pass"
 ALPHA_WALLET_SEED="b280922d2cffda44648346412c5ec97f429938105003730414f10b01e1402eac"
 BETA_WALLET_SEED="aabbcaabbcaabbcaabbcaabbcaabbcaabbcaabbcaabbcaabbcaabbcaabbcaabbc"
-WALLET_PASS=123
+WALLET_PASS=abc
```

Comment:
This looks really handy.

One thing I had shashed for harness.sh was this:

```diff
@@ -103,6 +103,11 @@ chmod +x "${NODES_ROOT}/harness-ctl/reorg"
 # Shutdown script
 cat > "${NODES_ROOT}/harness-ctl/quit" <<EOF
 #!/bin/sh
+tmux send-keys -t $SESSION:3 C-c
+tmux send-keys -t $SESSION:4 C-c
+tmux send-keys -t $SESSION:5 C-c
+tmux send-keys -t $SESSION:6 C-c
+sleep 0.2
 tmux send-keys -t $SESSION:1 C-c
 tmux send-keys -t $SESSION:2 C-c
 tmux wait-for alphadcr
```

I didn't understand how we were getting away with only interrupting the alpha and beta dcrd procs, but not the dcrwallets.